### DISCUSSION TITLE
Is it possible to chain patterns in Entity Ruler?

### DISCUSSION THREAD

**Discussion by clasnake** [explosion/spaCy#8182] [Help: Coding & Implementations] [answered]
For example, how to specify a pattern in a Ruler to match the LEMMA of a token which is in LOWER as well?

**Comment by polm** [comment, marked as answer]
Using Matcher syntax you can specify multiple conditions for a single token by using a single dictionary, so for example... 

    pattern = [{"LIKE_NUM": True, "LENGTH": 3}] 

would match three-character number like tokens. 

Note that lemmas should always be lower case though, except for maybe proper nouns. 

If that doesn't solve your problem could you give a more specific example of what you're trying to do?
